client/lib/cart-values/cart-items.js

```diff
@@ -395,6 +399,13 @@ function domainRegistration( properties ) {
 	return assign( domainItem( properties.productSlug, properties.domain ), { is_domain_registration: true } );
 }
 
+domainRegistration.fromSuggestion = function( suggestion ) {
```

Comment:
I don't think this method should be added to this file - it seems only useful in a very narrow context. Maybe you could just call `domainRegistration` directly [here](https://github.com/Automattic/wp-calypso/pull/6075/files#diff-0783639df235f2e67990963998ab21d1R67) instead of adding a function to a function?